core/src/com/ericc/the/game/map/TileTexturer.java

```diff
@@ -0,0 +1,206 @@
+package com.ericc.the.game.map;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.ericc.the.game.Enums;
+
+import java.util.HashMap;
+import java.util.Random;
+
+public class TileTexturer {
+    private Texture dunVoid;
+    private Texture floorMid1, floorMid2;
+    private Texture wallU1, wallU2, wallU3, wallU4;
+    private Texture wallD1, wallD2;
+    private Texture wallL1, wallL2, wallL3;
+    private Texture wallR1, wallR2, wallR3;
+    private Texture wallLU, wallRU, wallLD, wallRD;
+    private Texture wallLUcorner, wallLDcorner, wallRUcorner, wallRDcorner,
+            wallFancyCornerSmallLeft, wallTwoCorners, wallFancyCornerSmallRight;
+    private Texture wallCorridorMiddle, wallCorridorUp;
+    private HashMap<String, Enums.TILETEXTURE> textures;
+    private Random random;
+
+    TileTexturer() {
+        this.textures = new HashMap<String, Enums.TILETEXTURE>();
+        this.random = new Random();
+
+        loadTextures();
+        generateTextures();
+    }
+
+    private void loadTextures() {
+        dunVoid = new Texture("void.png");
+        floorMid1 = new Texture("mid_dun_flr1.png");
+        floorMid2 = new Texture("mid_dun_flr2.png");
+        wallD1 = new Texture("d_dun_wall1.png");
+        wallU1 = new Texture("u_dun_wall1.png");
+        wallU2 = new Texture("u_dun_wall2.png");
+        wallU3 = new Texture("u_dun_wall3.png");
+        wallU4 = new Texture("u_dun_wall4.png");
+        wallL1 = new Texture("l_dun_wall1.png");
+        wallR1 = new Texture("r_dun_wall1.png");
+        wallLD = new Texture("ld_dun_wall.png");
+        wallRD = new Texture("rd_dun_wall.png");
+        wallLU = new Texture("lu_dun_wall.png");
+        wallRU = new Texture("ru_dun_wall.png");
+        wallLUcorner = new Texture("left_up_corner.png");
+        wallLDcorner = new Texture("left_down_corner.png");
+        wallRUcorner = new Texture("right_up_corner.png");
+        wallRDcorner = new Texture("right_down_corner.png");
+        wallCorridorMiddle = new Texture("middle.png");
+        wallCorridorUp = new Texture("corridor_up.png");
+        wallFancyCornerSmallLeft = new Texture("corner_fancy.png");
+        wallTwoCorners = new Texture("two_corners.png");
+        wallFancyCornerSmallRight = new Texture("corner_fancy_right.png");
```

Comment:
przydałoby się wyjaśnienie, bo dla mnie to nie jest w ogóle jasne jakim kształtem jest fancy corner small right